ipa-core/src/config.rs

```diff
@@ -1052,21 +1057,21 @@ DwEB/wQEAwICpDAdBgNVHSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwCgYIKoZI
 zj0EAwIDSAAwRQIhAOTSQWbN7kfIatNJEwWTBL4xOY88E3+SOnBNExCsTkQuAiBB
 /cwOQQUEeE4llrDp+EnyGbzmVm5bINz8gePIxkKqog==
 -----END CERTIFICATE-----
-\"\"\"
-url = \"helper3.prod.ipa-helper.shard0.dev:443\"
-shard_port = 666
+"""
+url = "helper3.shard0.org:443"
+shard_url = "helper3.shard0.org:666"
```

Comment:
love the port - DOOM was my favourite game when I was a teenager

Reply:
lol 😈